stock_customer_deposits/readme/USAGE.md

```diff
@@ -0,0 +1,38 @@
+To use this module, you need to:
+
+Create Customer deposits
+
+1. Go to Sales > Quotations.
+2. Create new quotation to the desired customer
+3. Add storable product in order lines.
+4. Select route **Customer Deposit** in line.
+5. Add quantity you will keep as a deposit.
+6. Confirm quotation.
+7. Two pickings will be created.
+8. Validate first picking.
+9. Now you could invoice the order
+9. Access the second delivery note, the owner has been automatically assigned.
+10. Select the location where you want to deposit the customer's stock in your warehouse. It can be the same location or a different one.
+
+View Customer deposits
+
+11. Go to Inventory > Reporting > Locations.
+12. Filter the owner by the customer you created the order.
+13. Go to Sales > Customers.
+14. A smart button with deposits is shown if customer has deposits in some warehouse.
+
+Deliver customer deposits
+
+15. Go to Sales > Quotation.
+16. Create a new quotation.
+17. Select customer who has a deposit in your warehouse.
+18. In page Other Info choose warehouse where deposit is located. (Only is multi-warehouse is activated)
+19. Smart button with deposits is displayed if the customer has deposits in the chosen warehouse.
+20. Add line with product in deposit.
+23. As a product in deposit, a button will appear green if you have enough in stock or grey if you do not have enough in stock.
+24. You can view the deposits for that product by clicking on the button.
+25. To use the stock in the deposit, you must check the Deposit in line.
```

Comment:
**question:** you mean "check" by clicking on the check mark? Or just making sure that there's enough qty in deposit?
![image](https://github.com/OCA/stock-logistics-workflow/assets/973709/2002c77b-7ccf-471e-81b6-61a8a769b899)

Reply:
Ah I can see now that it's the check box in the sale order line. However, in the previous step you're told to click on a button, so it's confusing. Please make this clearer...

![image](https://github.com/OCA/stock-logistics-workflow/assets/973709/a345645b-583c-4726-b4fb-79d0a02502ea)